app/src/main/AndroidManifest.xml

```diff
@@ -1076,6 +1076,18 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".ExampleOverviewActivity" />
         </activity>
+        <activity
+            android:name=".examples.style.TileJsonActivity"
```

Comment:
Do we need an equivalent for iOS? cc @macdrevx

Reply:
Yes, atm iOS is missing the tileset model. We will have to add that and back it up with this example

Reply:
Please ticket this if needed.